### PR TITLE
pr1

### PR DESCRIPTION
**Pull request by VladAndronik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# PR1
+
+```
+docker build --tag app-server:latest ./app-server/
```
Comment (truskovskiyk) [inline README.md:4]: fix this?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+```
+
+```
+docker pull 8092/app-server:latest
+docker run -it --rm -p 8000:8000 8092/app-server:latest
+```
+
+# PR2
```
Comment (truskovskiyk) [inline README.md]: fix this???

**Review by truskovskiyk — APPROVED** [review]
@VladAndronik looks good!

File: app-server/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
+
+WORKDIR /app
+COPY man.png /app
```
Comment (truskovskiyk) [inline app-server/Dockerfile:4]: why do you need this image?

Comment (VladAndronik) [reply app-server/Dockerfile:4]: just to use COPY command and being able to visualize something when server is up)

**Pull request merged**